### ISSUE TITLE
Syntax highlighting broken via `font-lock-flush`

### ISSUE THREAD

**Issue by gbgar** [joostkremers/pandoc-mode#61]
Hi Joost, commit  7f184c178d759e573eaed486d9d70c9b5c8c1eb0 breaks syntax highlighting for me (possibly due to running an old versions of emacs). I am running version 24.5.1 and `font-lock-flush` is unavailable.

Searching for what that function does, I stumbled across [this stack exchange post about fontification](http://emacs.stackexchange.com/questions/6073/how-to-add-more-fontify-rules)

which includes a bit of code that appears to choose between different font-locking functions:

```
(if (fboundp 'font-lock-flush) 
    (font-lock-flush) 
    (when font-lock-mode 
        (with-no-warnings (font-lock-fontify-buffer)))
```

Something like this could fix the problem, or perhaps a variable containing the preferred font-locking function could be added.

Thanks for your time!

**Comment by joostkremers** [comment]
Thanks for reporting this. Should be fixed in cb72eef.

**Comment by gbgar** [comment]
Thank you for fixing this so quickly -- I really appreciate it. Sorry for my delay closing this issue.
